src/openforms/appointments/contrib/qmatic/client.py

```diff
@@ -73,7 +77,7 @@ def QmaticClient() -> "Client":
     config = QmaticConfig.get_solo()
     assert isinstance(config, QmaticConfig)
```

Comment:
Should `cast` be used instead? Adds an extra import though, I guess this is just a personal preference

Reply:
no, @CharString has strong feelings about that :)

Reply:
I'm curious to know why :)

Reply:
`cast` is a lie and the runtime behaviour can differ from the declared type, while `assert` enforces runtime correctness too. If assert is causing too much overhead, we can also strip it out and run with `python -O`

basically `cast` is a last resort if there are really no better options (and I agree)